config/management/src/secure_backend.rs

```diff
@@ -132,6 +132,8 @@ impl TryInto<config::SecureBackend> for SecureBackend {
                     server,
                     ca_certificate: certificate,
                     token: Token::FromDisk(PathBuf::from(token)),
+                    // TODO(davidiw):  parse the network_timeout_ms from the parameter list.
+                    network_timeout_ms: 30_000,
```

Comment:
@davidiw It's not clear to me how this field should go "up" from here.  A follow on commit/PR is needed to get this to the point of editable by an operator.

Reply:
I don't think we need a todo here, I cannot imagine tweaks, but we always support adding optional parameters if we really want that.